Formula/bazel.rb

```diff
@@ -16,11 +16,14 @@ class Bazel < Formula
 
   def install
     ENV["EMBED_LABEL"] = "#{version}-homebrew"
+    # Force Bazel to put all temporary file in the homebrew cache
+    ENV["BAZEL_WRKDIR"] = "#{buildpath}"
```

Comment:
Done. Technically ./compile.sh create 2 subdir in BAZEL_WRKDIR but it is indeed cleaner to put it in a subdirectory.